otherlibs/sites_locations/src/helpers.ml

```diff
@@ -0,0 +1,169 @@
+module Location = struct
+  type t = string
+end
+
+module Section = struct
```

Comment:
Is it not possible to share this with dune somehow? Even something janky with `copy_file` woudld be preferable I think.